apps/protocol-frontend/src/components/ReportForm.tsx

```diff
@@ -109,95 +89,127 @@ const ReportForm = ({ onFinish }: { onFinish: () => void }) => {
     setUserCreatingMore(!isUserCreatingMore);
   };
 
+  const userActivityTypesHook = useUserActivityTypesList();
+
+  // renaming these on destructuring incase we have parallel queries:
+  const {
+    isLoading: userActivityTypesIsLoading,
+    isFetching: userActivityTypesIsFetching,
+    isError: userActivityTypesIsError,
+    data: userActivityTypesData,
+    error: userActivityTypesError, // unused for now -- handling globally
+  } = userActivityTypesHook;
```

Comment:
You can probably combine the destructuring and call but that comes down to preference

Reply:
Ah, great idea. I tend to bring them in and then do it, but destructuring upon call will be cleaner especially if/when we have parallel queries. Will be much more readable to do the destructure on call and we'd be able to more directly see the structures.